src/rgw/rgw_sal.h

```diff
@@ -378,6 +378,10 @@ class Bucket {
     virtual Attrs& get_attrs(void) { return attrs; }
     virtual int set_attrs(Attrs a) { attrs = a; return 0; }
     virtual int remove_bucket(const DoutPrefixProvider* dpp, bool delete_children, std::string prefix, std::string delimiter, bool forward_to_master, req_info* req_info, optional_yield y) = 0;
+    virtual int remove_bucket_bypass_gc(int concurrent_max, bool
```

Comment:
Is this a permanent api method?  I would have said that gc is an RGWRados internal

Reply:
Probably not.  It's used by admin ops currently.